app/models/qernel/initializer_inputs.rb

```diff
@@ -0,0 +1,19 @@
+module Qernel
+  class InitializerInputs
+    def self.for_area(area_code)
+      new(area_code).all
+    end
+
+    def initialize(area_code)
+      @area_attributes =
+        Etsource::Loader.instance.area_attributes(area_code)[:init] || {}
```

Comment:
I was initially against putting it in `graph.rb` because the file is already over 400 lines of code which is my opinion is too long for any class.  But you make a valid point. Maybe someday, the `graph.rb` needs some clean-up.